src/test/mediapipeflow_test.cpp

```diff
@@ -1900,12 +1901,14 @@ TEST_F(MediapipeConfigChanges, ConfigWithEmptyBasePath) {
     std::string graphPbtxtFileContent = pbtxtContent;
     std::string configFileContent = configFileWithEmptyBasePath;
     std::string configFilePath = directoryPath + "/config.json";
-    std::string graphFilePath = directoryPath + "/graph.pbtxt";
+    std::string graphFilePath = directoryPath + "/mediapipeGraph/graph.pbtxt";
 
     const std::string inputName{"in\""};
     const std::string newInputName{"in2\""};
 
     createConfigFileWithContent(configFileContent, configFilePath);
+    std::string defaultGraphDirectoryPath = directoryPath + "/mediapipeGraph";
+    mkdir(defaultGraphDirectoryPath.c_str(), 0777);
```

Comment:
You could use more c++'ish approach with
```suggestion
    std::filesystem::create_directories(directoryPath)
```